src/system/scheduler/scheduler.c

```diff
@@ -128,8 +135,11 @@ int scheduler_getNextProcess()
 
 void scheduler_killProcess(int processID)
 {
+	//TODO: this function has to be fully executed!!!
 	contexts[processID].state = PROCESS_TERMINATED;
-	contexts[processID].func = NULL;
+	contexts[processID].func = NULL; //TODO: set function which gets executed after a process has been killed
```

Comment:
Ensure that the process slot can be reused by the create_process handler and that it won't be scheduled again